framework/include/base/Assembly.h

```diff
@@ -737,6 +737,10 @@ class Assembly
   /// Temporary work vector to keep from reallocating it
   std::vector<dof_id_type> _temp_dof_indices;
 
+  /// Temporary work data for reinitAtPhysical()
+  std::vector<Point> _temp_reference_points;
+  FEType _temp_fe_type;
```

Comment:
Or, maybe you just meant to do [this]( https://github.com/jwpeterson/moose/commit/aa05a6f765a2f59102354a0d348730685819a141), which seems like a reasonable change.